app/pods/components/web-components/au-dropdown/item/template.hbs

```diff
@@ -1,5 +1,11 @@
 <li class="{{this.class}}">
-  <LinkTo class="auk-menu__item active {{this.skin}}" @route={{@route}} ...attributes>
-    {{yield}}
-  </LinkTo>
+  {{#if @route}}
+    <LinkTo class="auk-menu__item active {{this.skin}}" @route={{@route}} ...attributes>
+      {{yield}}
+    </LinkTo>
+  {{else}}
+    <a class="auk-menu__item active {{this.skin}}" href={{@href}} ...attributes>
+      {{yield}}
+    </a>
+  {{/if}}
```

Comment:
Deze heeft veel meer side-effects gemaakt dan je denkt.
Op heel veel plaatsen gebruiken we
```WebComponents::AuDropdown::Item``` zonder een @route of @href
Als die plaatsen gaan we nu ineens een `<linkTo>` veranderen naar een `<a>`

Reply:
Ik denk dat je dan beter de logica omdraait:
```
{{#if @href}}
 <a> ...
{{else}}
<LinkTo>
{{/if}}
```